src/styles/utils/_vars.scss

```diff
@@ -1 +1,8 @@
-$c-gray: #eee;
+$page-backgroun-color:#e4dfd3;
+$text-minor-color:#344f10;
+$menu-and-btn-background:#333;
+$left-title:#344f1080;
+$shop-link-color:#828282;
+$input-ivalid:#eb5757;
```

Comment:
There is a typo in the variable name. It should be $input-invalid instead of $input-ivalid.